newdle/client/src/selectors.js

```diff
@@ -98,15 +98,23 @@ export const newdleHasParticipantsWithoutEmail = createSelector(
   participants => participants.some(p => p.email === null)
 );
 export const getParticipantAvailability = createSelector(
+  getNewdle,
   getNewdleTimeslots,
   getNewdleParticipants,
-  (timeslots, participants) => {
-    return timeslots.map(timeslot => {
```

Comment:
Wouldn't `_.sortBy(..., x => x.startDt !== final_dt);` around the original code do the job as well?